### PR TITLE
Feature races

### PR DESCRIPTION
**Pull request by DanEvans485**
- added basic races for creating monsters (Not done traits for races)
- created seeds and updated character seeder

### CODE REVIEW

File: database/seeds/RaceSeeder.php [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+        /**
+         *
+         * Insert Monster Races
+         *
+         */
+        Race::create([
+            'name'     => 'Orc',
+            'code'     => 'orc',
+            'playable' => false,
```
Comment (G-Bro) [inline database/seeds/RaceSeeder.php]: Not entirely true. While orcs are not playable in '5e SRD' (the base rules as described in the player's handbook) they are playable in the Volo's Guide to Monsters expansion ruleset.

I think we might need to associate a race to a ruleset.

File: app/Models/Core/Race.php [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+namespace App\Models\Core;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Race extends Model
+{
+    protected $table = 'races';
+
+    public $timestamps = false;
+}
```
Comment (G-Bro) [inline app/Models/Core/Race.php]: Not really sure where to comment this but be aware that races applied to player character's provide certain attributes whereas a monster's race is literally just a string. We can probably just do something like:

````
if ($character->is_player) {
    applyRaceBonus();
}
````

File: database/seeds/RaceSeeder.php (hunk 1 above)
Comment (DanEvans485) [reply database/seeds/RaceSeeder.php]: I know but I just wanted some player and non player characters in my db.

Comment (DanEvans485) [reply database/seeds/RaceSeeder.php]: So if we had a rule set. You would have multiple (In this example) Orc races. Where they link to different rule sets.

File: app/Models/Core/Race.php (hunk 2 above)
Comment (DanEvans485) [reply app/Models/Core/Race.php]: Yes, I did say in the PR comment this was like bare min for creating monsters.

Comment (DanEvans485) [reply app/Models/Core/Race.php]: Can probs link to modifier table and apply the bonus that way?

File: database/seeds/RaceSeeder.php (hunk 1 above)
Comment (G-Bro) [reply database/seeds/RaceSeeder.php]: I'm thinking there could be two races, one monster_orc and one playable_orc

OR

the 'playable' attribute could come from the join to the ruleset

Comment (DanEvans485) [reply database/seeds/RaceSeeder.php]: Mmm I just think about when we have the case where a race (Or item or spell etc) doesn't exist in a ruleset like this little fucker https://www.dndbeyond.com/marketplace/source/locathah-rising .

**Pull request merged**